src/debugger/titaniumDebugAdapter.ts

```diff
@@ -1,5 +1,5 @@
 import { ProxyServer } from '@awam/remotedebug-ios-webkit-adapter';
-import { ChromeDebugAdapter, Crdp } from '@awam/vscode-chrome-debug-core';
+import { ChromeDebugAdapter, Crdp } from 'vscode-chrome-debug-core';
 import * as got from 'got';
 import { sleep } from '../common/utils';
 import { URL } from 'url';
```

Comment:
<!--
  0 failure: 
  1 warning:  src/debugger/tita...
  
  
  DangerID: danger-id-Danger;
  File: src/debugger/titaniumDebugAdapter.ts;
  Line: 5;
-->

- :warning: src/debugger/titaniumDebugAdapter.ts line 5 – Unexpected use of 'require("url").URL'. Use the global variable 'URL' instead. (node/prefer-global/url)